tools/summonerd/src/web.rs

```diff
@@ -30,7 +30,7 @@ async fn serve_summoning_jpg() -> impl IntoResponse {
         .status(StatusCode::OK)
         .header("Content-Type", "image/jpeg")
         .header("Cache-Control", "public, max-age=3600") // Cache for 1 hour
-        .body(axum::body::Full::from(jpg))
+        .body(axum::body::Body::from(jpg))
```

Comment:
`axum::body::Full` was removed, and one is to use `Body` instead (according to their changelog)